tests/test_utils.py

```diff
@@ -433,3 +434,12 @@ def test_get_ecosystem_from_manifest():
 
     resp = get_ecosystem_from_manifest("dependencies.txt")
     assert resp == "maven"
+
+
+def test_check_for_accepted_ecosystem():
+    """Test check_for_accepted_ecosystem function."""
+    resp = check_for_accepted_ecosystem("maven")
+    assert resp is True
```

Comment:
IMHO CI will fail on this line, `is` don't have to be there

Reply:
it doesnt fail. but changed anyways :)